assembl/static2/js/app/components/common/visibilityComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
It's a simple higher order component that displays its children only if a visibility prop given to it is true. It's the react equivalent of our CSS `hidden` class technique.